exla/mix.exs

```diff
@@ -33,7 +33,15 @@ defmodule EXLA.MixProject do
         priv_path = Path.join(Mix.Project.app_path(), "priv")
         cwd_relative_to_priv = relative_to(File.cwd!(), priv_path)
 
+        exla_cpu_only =
+          if System.get_env("EXLA_CPU_ONLY") in ["1", "true"] do
+            "true"
+          else
+            ""
+          end
+
         %{
+          "EXLA_CPU_ONLY" => exla_cpu_only,
```

Comment:
If we leave it out of here, the check should still work?

Reply:
I guess so. Originally I was checking for true under a different variable name, but now its only an ifdef